modules/ROOT/attachments/specs/enotices2-ted-europa-eu-esenders.yaml

```diff
@@ -77,14 +117,30 @@ paths:
                   type: string
                   format: binary
                 metadata:
-                  $ref: '#/components/schemas/EsenderAuthor'
+                  $ref: '#/components/schemas/NoticeAuthor'
```

Comment:
The current specification defines "EsenderAuthor", not "NoticeAuthor".

Reply:
I think Vangelis is right for this one too. you cannot change the request body of the current API.
We only want to improve the documentation part (only titles, summaries, descriptions etc).